getting_started_with_acceptance_tests.md

```diff
@@ -0,0 +1,80 @@
+# A Larger Product and Team
+At Ubiqua, as we started to grow both our product and engineering team, we realized some things had to be re-prioritezed and fixed. Some of the problems we were having were:
+
+- The onboarding of a single person to our software engineering team took more time than we desired. This was mainly being cause due to legacy code and the bad documentation of features and processes.
```

Comment:
I would find a way to shorten "The onboarding of a single person to our software engineering team", maybe: "Onboarding for new developers" or just "Onboarding"?

I think "being cause due to" should be rewritten as "being caused by"